src/sentry/api/endpoints/organization_teams.py

```diff
@@ -56,6 +56,8 @@ def validate(self, attrs):
 class OrganizationTeamsEndpoint(OrganizationEndpoint):
     permission_classes = (OrganizationTeamsPermission,)
 
+    team_serializer = TeamSerializer
```

Comment:
moved this serializer up to the class as well as the `should_add_creator_to_team` to facilitate differing behavior in the SCIM routes